src/main/kotlin/io/titandata/titan/providers/local/Migrate.kt

```diff
@@ -101,7 +101,7 @@ class Migrate (
         val updateRepo = Repository(name, metadata)
         repositoriesApi.updateRepository(name, updateRepo)
         docker.run(image, "", arguments)
-        commit(name, "Initial Migration")
+        commit(name, "Initial Migration", listOf())
```

Comment:
` emptyList<String>()` is probably better than `listOf()` because it declares the type and that it's empty.